### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by rahulcs754**
Text Utilities
-> headings
-> small text
-> gray text
-> center text

image

image

image

Please check and review

### CODE REVIEW

**Comment by RahulSingh9131** [comment]
everything looks good to me . you can go ahead.

File: css/utils/utilities.css [hunk 1]
```diff
@@ -264,6 +264,58 @@ margin-left: 1rem;
...
+
+.lh-sm {
+  line-height: 1.25;
+}
+
+.lh-large {
+  line-height: 1.6;
+}
+
+
```
Comment (shubhamkumar648) [inline css/utils/utilities.css]: remove space

File: css/utils/utilities.css [hunk 2]
```diff
@@ -264,6 +264,58 @@ margin-left: 1rem;
...
+}
+
+.text-uppercase {
+  text-transform: uppercase;
+}
+
+.text-capitalize {
+  text-transform: capitalize;
+}
+
```
Comment (shubhamkumar648) [inline css/utils/utilities.css:291]: same here

**Review by shubhamkumar648 — COMMENTED** [review]
cheack changes

File: css/utils/utilities.css (hunk 1 above)
Comment (rahulcs754) [reply css/utils/utilities.css]: done

File: css/utils/utilities.css (hunk 2 above)
Comment (rahulcs754) [reply css/utils/utilities.css:291]: done

**Review by shubhamkumar648 — APPROVED** [review]
good to merge

**Review by rittik-ghosh — COMMENTED** [review]
Looks good to merge

**Pull request merged**